### ISSUE TITLE
[Feature Request]: Find a way to disable the remote fetches for fonts and the iframe resizer

### ISSUE THREAD

**Issue by ilikenwf** [anapnoe/stable-diffusion-webui-ux#82]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do ?

Gradio pulls from google and cloudflare which is an issue for local installs on non internet machines. In addition it's a privacy problem because Google is generally evil. 

### Proposed workflow

It would be nice to have non-CDN versions of the fonts and iframe resizer by default.

**Comment by anapnoe** [comment]
Yes we can do that the issue is with gradio components that aren't included on the startup and are part of various extensions but I think we can create a bundle with all of them

**Comment by halr9000** [comment]
@anapnoe guess what I found when profiling my startup delay issues? Was about to add to the issue I opened but look at where this delay is...related.

![image](https://github.com/anapnoe/stable-diffusion-webui-ux/assets/18482/f6ed8b2b-3aab-42d8-9c38-680165b5a6ff)
